src/share.js

```diff
@@ -60,22 +57,70 @@ class Share extends BasePlugin {
     if (!this.config.shareUrl) {
       this.config.shareUrl = window.location.href;
     }
-
+    this._wasPlayed = false; // keep state of the player so we can resume if needed
```

Comment:
Should plugin reset value for `this._wasPlayed` on player reset hook?

Reply:
Right, fixed, thank 
BTW this refactor is inspired by info-plugin, grate job there